yapio.c

```diff
@@ -1349,6 +1384,49 @@ yapio_blk_md_t *
 yapio_test_ctx_to_md_array(const yapio_test_ctx_t *,
                            enum yapio_test_ctx_mdh_in_out, int *);
 
+/* Picks out which IOs will be skipped */
+static void
+select_skipped_ios(yapio_test_ctx_t *ytc)
+{
+    /* number of IO operations by this rank */
+    int num_ops;
+    yapio_test_ctx_to_md_array(ytc, YAPIO_TEST_CTX_MDH_IN, &num_ops);
+
+    printf("numops = %d\n", num_ops);
+
+    /* TODO need to YAPIO_FREE elsewhere */
+    bool *skipped_ios = YAPIO_MALLOC(num_ops * sizeof(bool));
```

Comment:
This is going to allocate a lot of memory when the number of blocks is large.  Since we already have array items for each block is this really needed?

Reply:
+1

Reply:
:+1: